daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/ValueTranslator.scala

```diff
@@ -23,7 +22,7 @@ private[engine] final class ValueTranslator(compiledPackages: CompiledPackages)
   // this is not tail recursive, but it doesn't really matter, since types are bounded
   // by what's in the source, which should be short enough...
   @throws[PreprocessorException]
-  private def replaceParameters(params: ImmArray[(TypeVarName, Type)], typ0: Type): Type =
+  private[this] def replaceParameters(params: Traversable[(TypeVarName, Type)], typ0: Type): Type =
```

Comment:
```suggestion
  private[this] def replaceParameters(params: Iterable[(TypeVarName, Type)], typ0: Type): Type =
```
scala 2.13 killed traversable